api/queries_pr.go

```diff
@@ -629,17 +631,55 @@ func CreatePullRequest(client *Client, repo *Repository, params map[string]inter
 	return pr, nil
 }
 
-func UpdatePullRequestReviews(client *Client, repo ghrepo.Interface, params githubv4.RequestReviewsInput) error {
-	var mutation struct {
-		RequestReviews struct {
-			PullRequest struct {
-				ID string
-			}
-		} `graphql:"requestReviews(input: $input)"`
+// AddPullRequestReviews updates the requested reviewers on a pull request using the REST API.
+func AddPullRequestReviews(client *Client, repo ghrepo.Interface, prNumber int, users, teams []string) error {
+	if len(users) == 0 && len(teams) == 0 {
+		return nil
 	}
-	variables := map[string]interface{}{"input": params}
-	err := client.Mutate(repo.RepoHost(), "PullRequestUpdateRequestReviews", &mutation, variables)
-	return err
+
+	path := fmt.Sprintf("repos/%s/%s/pulls/%d/requested_reviewers", repo.RepoOwner(), repo.RepoName(), prNumber)
```

Comment:
We should apply `url.PathEscape`:


```suggestion
	path := fmt.Sprintf("repos/%s/%s/pulls/%d/requested_reviewers", url.PathEscape(repo.RepoOwner()), url.PathEscape(repo.RepoName()), prNumber)
```